etl/steps/data/garden/wb/2024-12-03/poverty_projections.py

```diff
@@ -0,0 +1,125 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+import owid.catalog.processing as pr
+from owid.catalog import Table
+from owid.datautils.dataframes import map_series
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+# Define latest year without projections
+LATEST_YEAR_WITHOUT_PROJECTIONS = 2024
+
+# Define tables to be loaded.
+TABLES = ["country", "region", "global"]
+
+# Define scenarios and new names
+SCENARIOS = {
+    "historical": "Historical",
+    "current_forecast": "Current forecast + historical growth",
+    "2pct": "2% growth",
+    "2pct_gini1": "2% growth + Gini reduction 1%",
+    "2pct_gini2": "2% growth + Gini reduction 2%",
+    "4pct": "4% growth",
+    "6pct": "6% growth",
+    "8pct": "8% growth",
+}
+
+# Define index columns
+INDEX_COLUMNS = ["country", "year", "povertyline", "scenario"]
+
+# Define indicator columns
+INDICATOR_COLUMNS = ["fgt0", "poorpop"]
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow = paths.load_dataset("poverty_projections")
+
+    # Read table from meadow dataset.
+    # Define empty table list to store tables.
+    tables = []
+    for table in TABLES:
+        tb = ds_meadow.read(table)
+
+        # Append table to list.
+        tables.append(tb)
```

Comment:
if you want to read all tables in `ds_meadow` (which I think is what you do here), you can simply do:

```suggestion
    # Read tables from meadow dataset.
    tables = list(ds_meadow)
```

Reply:
Ah, that's good to know, thanks! I don't think I will use it here, because I won't use the country table in Garden. I will correct that

Reply:
@paarriagadap If you want to use a subset of tables like in here, you could use a one-liner instead:

```python
tables = [ds_meadow.read(table_name) for table_name in TABLES]
```

Reply:
That's true, thank you!